upgrades/v1.0.0.md

```diff
@@ -0,0 +1,48 @@
+# Kroma v1.0.0 Upgrade Instructions
+
+This document provides instructions to upgrade from Kroma v0.3.0 to v1.0.0.
+
+## Guides for upgrade
+
+### Stop Kroma
+
+For upgrade, stop your Kroma full node or validator.
+```bash
+# for full node
+docker compose --profile fullnode down -v
+
+# for validator
+docker compose --profile validator down -v
```

Comment:
I think removing the `-v` option will prevent users from having their volumes removed.

Reply:
I see. I'll remove it.